DependencyInjection/Factory/Loader/FileSystemLoaderFactory.php

```diff
@@ -61,4 +63,18 @@ public function addConfiguration(ArrayNodeDefinition $builder)
                 ->end()
             ->end();
     }
+
+    /**
+     * @param string $reference
+     *
+     * @return Reference
+     */
+    private function createLocatorReference($reference)
+    {
+        return new Reference(
+            sprintf('liip_imagine.binary.locator.%s', $reference),
+            ContainerInterface::EXCEPTION_ON_INVALID_REFERENCE,
+            SymfonyFramework::hasDefinitionSharedToggle()
```

Comment:
Ok; ammended the code to the following:
```php
$name = sprintf('liip_imagine.binary.locator.%s', $reference);

if (SymfonyFramework::hasDefinitionSharedToggle()) {
    return new Reference($name);
}

return new Reference($name, ContainerInterface::EXCEPTION_ON_INVALID_REFERENCE, false);
```

I didn't use a version toggle, as I want to use the exact mechanism that requires a non-strict reference, which [can be found here](https://github.com/liip/LiipImagineBundle/pull/875/files#diff-761cf6b460d040d983ac2342a13ff0c5R35).